lib/install/tailwind/tailwind.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   mode: 'jit',
```

Comment:
JIT is now the default mode https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.0.0 so we can also ✂️ this line 

```suggestion

```